docs/getting-started/proxies/traefik.md

```diff
@@ -55,6 +55,9 @@ To run PhotoPrism behind Traefik, create a `traefik.yaml` configuration and then
               scheme: https
       websecure:
         address: ":443"
+        transport:
+          respondingTimeouts:
+            readTimeout: "0s"
```

Comment:
How about we set it to the following, as at least a long read timeout might be desirable (unless it would also conflict with websockets, for example)?

```yaml
      respondingTimeouts:
        readTimeout: "1h"
        writeTimeout: "0s"
        idleTimeout: "300s"
```

Reply:
Great idea, makes sense to supply high-enough defaults. 👍🏻

Reply:
```suggestion
            readTimeout: "1h"
            writeTimeout: "0s"
            idleTimeout: "300s"
```